README.md

```diff
@@ -9,106 +9,236 @@
 - Document how a component behaves in various scenarios.
 - Get a11y, snapshot and visual tests using other addons for free.
 
+[Full documentation and live demos](https://msw-sb.vercel.app/)
+
+## Installing and Setup
+
+### Install MSW and the addon
+
+```sh
+npm i -D msw msw-storybook-addon
+```
+
+### Generate service worker for MSW in your public folder.
+
+_If you already use MSW in your project, you have likely done this before so you can skip this step._
+
+```sh
+npx msw init public/
+```
+
+Refer to the [MSW official guide](https://mswjs.io/docs/getting-started/integrate/browser) for framework specific paths if you don't use `public`.
+
+### Enable MSW in Storybook by adding this to `./storybook/preview.js`
```

Comment:
This heading looks more like a paragraph that should go below it. What do you think if we rephrase it as `### Configure Storybook`/`Add the add-on`?

Reply:
good one!